src/webdriver.nim

```diff
@@ -33,6 +33,9 @@ type
   ProtocolException* = object of WebDriverException
   JavascriptException* = object of WebDriverException
 
+let
+  defaultSessionCapabilities = %*{"capabilities": {"browserName": "firefox"}}
+
```

Comment:
I wanted to make this `const` but the compiler doesn't seem to like a `const JsonNode`, so I used `let` instead. If there's a way to make this `const`, let me know; that seems more appropriate.

Reply:
LGTM. Only thing I can suggest is to export this so that it's shown in the docgen.

Reply:
Do you mean mark it public? Do you want me to add some kind of additional comments to it?

Reply:
nah, feel free to just stick a `*` in there :)